app/views/repos/_script.html.haml

```diff
@@ -1,9 +1,10 @@
-- if script_url = benchmark_type.script_url
+- if script_url = benchmark.script_url
   %h2.l-align-center
     = t('repos.script_title', graph_name: benchmark_name)
 
-    = link_to benchmark_type.github_url, class: 'btn btn-primary', target: '_link' do
+    = link_to benchmark.github_url, class: 'btn btn-primary', target: '_link' do
       = t('repos.view_github')
 
   - formatter = Rouge::Formatters::HTMLPygments.new(Rouge::Formatters::HTML.new)
+  - require 'net/http' unless defined?(Net::HTTP)
```

Comment:
I had an issue with net/http locally saying that Net::HTTP is not recognized, although not having the same in production and test environment. So this was a quick workaround. 

Do you have any idea if I can maybe avoid this require? Since it's needed only in dev environment.

Where should I place `require`'s, is there any good practice?